app/pages/home/homePageController.ts

```diff
@@ -25,22 +28,30 @@ export class HomePageController {
   };
 
   public update() {
+
+    // TODO: Remove these try/catches once we are not relying on 'additionalServers' in config.js
```

Comment:
@spadgett opened an issue in common to track it, watch is a problem still https://github.com/openshift/origin-web-common/issues/17